examples/comparisons/compare_embeddings.py

```
@@ -0,0 +1,161 @@
+
+from __future__ import annotations
+
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from umap import UMAP
+
+from elementembeddings.compare import (
```

Flake8 (7.3.0)

[error] 'elementembeddings.compare._get_common_elements' imported but unused

(F401)